package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A WebdriverIO plugin. Helper module to run WebdriverIO commands synchronously.",
   "main": "build/index.js",
   "scripts": {
-    "test": "grunt"
+    "test": "./node_modules/.bin/grunt"
```

Comment:
as far as I know this is not necessary in the package.json .. npm automatically uses the binary in node_modules